packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2497,3 +2516,31 @@ const findBookingQuery = async (bookingId: number) => {
   // Don't leak any sensitive data
   return foundBooking;
 };
+
+function isLoggingEnabled({
+  eventTypeId,
+  usernameOrTeamName,
+}: {
+  eventTypeId?: number | null;
+  usernameOrTeamName?: string | string[];
+}) {
+  const usernameOrTeamnamesList =
+    usernameOrTeamName instanceof Array ? usernameOrTeamName : [usernameOrTeamName];
+  // eslint-disable-next-line turbo/no-undeclared-env-vars
+  const bookingLoggingEventIds = process.env.BOOKING_LOGGING_EVENT_IDS || "";
```

Comment:
Will add it to .env.example and turbo.json once we finalize this is what we need